### PR TITLE
Support multiple roles paths

### PR DESCRIPTION
**Pull request by ehelms**
##### ISSUE TYPE

 - Feature Pull Request

##### SUMMARY
Allow specifying multi roles paths through the command line or conductor settings.

### CODE REVIEW

**Comment by chouseknecht** [comment]
Looks good.

Some updates to docs would be much appreciated. :-)

File: container/templates/build-docker-compose.j2.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ ansible-container:
     - {{ base_path }}:/ansible-container/{% if params.selinux %}:Z{% endif %}
     {% if params.with_volumes %}{% for vol in params.with_volumes %}
     - {{ vol }}{% endfor %}{% endif %}
-    {% if params.roles_path %}- {{ params.roles_path }}:/local-roles{% endif %}
+    {% for role_path in params.roles_path %}
+    - {{ role_path }}:/local-roles/{{ role_path }}
+    {% endfor %}
   working_dir: /ansible-container/ansible/
```
Comment (chouseknecht) [inline container/templates/build-docker-compose.j2.yml]: We don't actually use this template anymore. I guess it's OK to update it, but we really should just get rid of it.

File: container/templates/listhosts-docker-compose.j2.yml [hunk 2]
```diff
@@ -21,7 +21,9 @@ ansible-container:
     - {{ base_path }}:/ansible-container/{% if params.selinux %}:Z{% endif %}
     {% if params.with_volumes %}{% for vol in params.with_volumes %}
     - {{ vol }}{% endfor %}{% endif %}
-    {% if params.roles_path %}- {{ params.roles_path }}:/local-roles{% endif %}
+    {% for role_path in params.roles_path %}
+    - {{ role_path }}:/local-roles/{{ role_path }}
+    {% endfor %}
   working_dir: /ansible-container/ansible/
```
Comment (chouseknecht) [inline container/templates/listhosts-docker-compose.j2.yml]: Same with this template. It's no longer used.

**Comment by ehelms** [comment]
Docs have been updated to reflect new option style. Template removal -- https://github.com/ansible/ansible-container/pull/668

**Comment by chouseknecht** [comment]
LGTM

**Pull request merged**